pkg/asset/installconfig/basedomain.go

```diff
@@ -36,8 +36,10 @@ func (a *baseDomain) Generate(parents asset.Parents) error {
 	switch platform.CurrentName() {
 	case aws.Name:
 		var err error
-		a.BaseDomain, err = awsconfig.GetBaseDomain()
+		zone, err := awsconfig.GetBaseDomain()
 		cause := errors.Cause(err)
+		a.BaseDomain = zone.Name
+		platform.AWS.PublicZoneID = zone.ID
```

Comment:
this can panic if the return zone is `nil`

Reply:
Fixed.